filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,112 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
+  license: CC-0
+  xref:
+    wikidata: Q389314
+doc: |
+  SquashFS is a compressed filesystem in a file, that can be read-only
+  accessed from low memory devices. It is popular for booting LiveCDs and
+  packing self-contained binaries. SquashFS format is used by Ubuntu .snap
+  packages. SquashFS is natively supported by Linux Kernel.
+doc-ref: https://github.com/AgentD/squashfs-tools-ng/blob/master/doc/format.txt
+seq:
+  - id: superblock
+    type: superblock
```

Comment:
Why not move the `superblock` definition of `seq` directly here?

Reply:
I guess because this spec should be called `squashfs`, not `squashfs_superblock`.

Reply:
Yep. It should. But for now I am unable to overcome the complexity to develop it further.